src/helper/Validation.js

```diff
@@ -0,0 +1,57 @@
+noContent = footsteps => {
+  let valid = false
+
+  for (var i = 0; i < footsteps.length; i++) {
```

Comment:
Avoid using var. It leads to memory leak issues. use let 

`  for (let i = 0; i < footsteps.length; i++) {
`